exercises/launchintro-execpols.cpp

```diff
@@ -242,8 +242,8 @@ int main(int RAJA_UNUSED_ARG(argc), char **RAJA_UNUSED_ARG(argv[]))
 
 // _raja_tensorinit_cuda_start
   using cuda_teams_z_3 = RAJA::expt::LoopPolicy<RAJA::cuda_block_z_direct>;
-  using cuda_global_thread_y_3 = RAJA::expt::LoopPolicy<RAJA::cuda_global_thread_y>;
-  using cuda_global_thread_x_3 = RAJA::expt::LoopPolicy<RAJA::cuda_global_thread_x>;
+  using cuda_global_thread_y_3 = RAJA::expt::LoopPolicy<RAJA::expt::cuda_global_thread_y>;
```

Comment:
Is this an issue in develop too?

Reply:
I mentioned this in the PR description under "Other Changes", the invisible expt alias breaks when you have namespaces that look like RAJA::expt::N and RAJA::N. MSVC in particular doesn't know how to handle this so the expt namespace alias has been removed.

Reply:
Going to wait on #1273 to address this.